build-tools/packages/build-tools/src/fluidBuild/fluidBuild.ts

```diff
@@ -21,15 +22,20 @@ parseOptions(process.argv);
 async function main() {
 	const timer = new Timer(commonOptions.timer);
 	const resolvedRoot = await getResolvedFluidRoot(true);
-
-	log(`Build Root: ${resolvedRoot}`);
+	const fluidConfig = getFluidBuildConfig(resolvedRoot, false);
+	const isDefaultConfig = fluidConfig === DEFAULT_FLUIDBUILD_CONFIG;
+	const suffix = isDefaultConfig
+		? ` (${chalk.yellowBright("inferred packages and tasks")})`
+		: "";
+	log(`Build Root: ${resolvedRoot}${suffix}`);
 
 	// Load the packages
 	const repo = new FluidRepoBuild({
 		repoRoot: resolvedRoot,
 		gitRepo: new GitRepo(resolvedRoot),
-		fluidBuildConfig: getFluidBuildConfig(resolvedRoot),
+		fluidBuildConfig: getFluidBuildConfig(resolvedRoot, false),
```

Comment:
any reason to fetch again? seems like the config shouldn't have changed
```suggestion
		fluidBuildConfig: fluidConfig,
```